docs/housekeeping-git-traffic.html

```diff
@@ -120,7 +120,7 @@ <h3>Transferred Data</h3>
 		}'></canvas>
 	<div class="info-box">
 		<p>
-			The amount of data transferred for clones and fetches.
+			The amount of data transferred for clones and fetches (<b>not</b> including <a href="https://git-lfs.github.com/">Git LFS</a> traffic).
```

Comment:
I’d `<em>` this instead of making the text bold. (Note that I generally prefer `<em>` over `<i>` and `<strong>` over `<b>`, because `<i>` and `<b>` imply a certain style and not structure, which is something CSS should take care of and not HTML.)